packages/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake

```diff
@@ -44,6 +45,11 @@ file(GLOB input_SRC CONFIGURE_DEPENDS
         ${REACT_ANDROID_DIR}/cmake-utils/default-app-setup/*.cpp
         ${BUILD_DIR}/generated/autolinking/src/main/jni/*.cpp)
 
+# Ensure that `input_SRC` paths use forward slashes
+foreach(path IN LISTS input_SRC)
+    string(REPLACE "\\" "/" path "${path}")
+endforeach()
+
```

Comment:
- Problem: The variables REACT_ANDROID_DIR and BUILD_DIR might also contain backslashes if they are set on a Windows system. These backslashes need to be replaced to avoid similar issues.

- Solution: Check if the script is running on Windows using if(CMAKE_HOST_WIN32). If it is, replace backslashes with forward slashes for these variables as well: